templates/d/buildwasm.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+make clean
+make
+tic80 --fs . --cmd 'load wasmdemo.wasmp & import binary cart.wasm & run & exit'
```

Comment:
Just me, but nothing about `build` to me implies "run" also...  I think perhaps we need `build_and_run` or something...? Thoughts?

Reply:
I'm fine calling it ```build_and_run```. As is typical, this started out with the make commands, and then I added the tic80 command line too to run. As the README states, these are just convenience scripts.